haproxy.tf

```diff
@@ -68,23 +92,17 @@ containerPorts:
 existingConfigmap: haproxy
 ingress:
   enabled: true
+  ingressClassName: nginx
   hostname: ${var.relay_name}.${var.primary_base_domain}
   annotations:
+    ${indent(4, local.cert_manager_annotations_yaml)}
+    ${indent(4, local.dns_annotations_yaml)}
     nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
-    cert-manager.io/cluster-issuer: "letsencrypt"
-    nginx.ingress.kubernetes.io/app-root: /enr
-    cert-manager.io/issue-temporary-certificate: "true"
-    acme.cert-manager.io/http01-edit-in-place: "true"
-    nginx.org/server-snippets: |
-      location ~* ^/(?!enr$|enr/|$) {
-        return 403;
-      }
-    nginx.ingress.kubernetes.io/configuration-snippet: |
-      location ~* ^/(?!enr$|enr/|$) {
-        return 403;
-      }
```

Comment:
we losing these server snippet blocks?

Reply:
This was very restrictive and caused the cert-manager acme records validation to fail (hashquard faced this issue when migrating).
Also, the p2p relays do not expose go endpoints